services/licenseStore.test.js

```diff
@@ -127,19 +127,43 @@ describe('licenseStore', () => {
   });
 
   describe('compatible()', () => {
-    it('finds no compatible licences for "cc by-sa 4.0"', () => {
-      expect(subject.compatible('cc-by-sa-4.0')).toEqual([]);
+    const expectedKeys = ['id', 'name', 'groups', 'compatibility', 'regexp', 'url'];
+
+    it('returns an array of licenses', () => {
+      const compatible = subject.compatible('CC BY-SA 3.0');
+      compatible.forEach(license => {
+        expect(Object.keys(license)).toEqual(expectedKeys);
+      });
+    });
+
+    it('finds compatible licenses for "CC BY-SA 3.0"', () => {
+      const compatible = subject.compatible('CC BY-SA 3.0');
+      expect(compatible.map(license => license.id)).toEqual(['cc-by-sa-3.0-de', 'cc-by-sa-4.0']);
+    });
+
+    it('finds no compatible licences for "CC BY-SA 4.0"', () => {
+      const compatible = subject.compatible('CC BY-SA 4.0');
+      expect(compatible).toEqual([]);
     });
```

Comment:
we probably need a test for `const compatible = subject.compatible('not a licence');`

Reply:
Added this test below 👇